src/Tizen.NUI/src/public/BaseComponents/ControlState.cs

```diff
@@ -235,7 +235,7 @@ public override string ToString()
         /// <param name="rhs">A <see cref="ControlState"/> on the right hand side.</param>
         /// <returns>true if the ControlStates are not equal; otherwise, false.</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public static bool operator !=(ControlState lhs, ControlState rhs) => !lhs.Equals(rhs);
+        public static bool operator !=(ControlState lhs, ControlState rhs) => lhs == null || !lhs.Equals(rhs);
```

Comment:
In case that both `lhs` and `rhs` are null, this will return true. Is it OK?

Reply:
When lhs and rhs are null, `operator !=` should return false.